src/missed_spans.rs

```diff
@@ -144,8 +144,14 @@ impl<'a> FmtVisitor<'a> {
                     line_start = offset + subslice.len();
 
                     if let Some('/') = subslice.chars().skip(1).next() {
-                        // Add a newline after line comments
-                        self.buffer.push_str("\n");
+                        // check that there are no contained block comments
+                        if !subslice.chars()
+                            .filter(|&c| c != '\t' && c != ' ')
+                            .collect::<String>()
```

Comment:
could this be done more efficiently? Am I right that you only want to trim leading whitespace? So you should be able to use trim and then contains, without needing to collect?